crates/cxx-qt-gen/src/generator/cpp/constructor.rs

```diff
@@ -0,0 +1,128 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
don't forget to add info about the constructor trait to the book :-)

Reply:
Like with the Changelog I think we should post-pone this until the feature is more stable.
Let's keep it as a "hidden" feature for now.